app/src/reducers/index.js

```diff
@@ -18,4 +19,5 @@ export default combineReducers({
   profile,
   welcome,
   walkthrough,
+  onboarding
```

Comment:
I'd rename this to "isOnboarded" to be more explicit of what it is
```suggestion
  isOnboarded,
```